test/DebugInfo/DebugTypeMember.ll

```diff
@@ -0,0 +1,63 @@
+; Exercise Value field in SPIRV TypeMember.
```

Comment:
is this test in principle different from https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/main/test/DebugInfo/NonSemantic/static_member_array.ll ?

Reply:
The unique part of this test is that it has an DIFlagStaticMember and extraData field.  This exercises code in LLVMToSPIRVDbgTran.cpp lines 1016 and 1045.